ArduPlane/quadplane.cpp

```diff
@@ -716,6 +718,12 @@ bool QuadPlane::setup(void)
     motors->set_update_rate(rc_speed);
     attitude_control->parameter_sanity_check();
 
+    // Try to convert mot PWM params, if still invalid force conversion
+    AP_Param::convert_old_parameters(&mot_pwm_conversion_table[0], ARRAY_SIZE(mot_pwm_conversion_table));
+    if ((motors->get_pwm_output_min() <= 0) || (motors->get_pwm_output_max() <= 0) || (motors->get_pwm_output_max() <= motors->get_pwm_output_min())) {
```

Comment:
With https://github.com/ArduPilot/ardupilot/pull/20900 this could be changed to:
```suggestion
    if (!motors->check_mot_pwm_params()) {
```

Reply:
done